src/viewer/main/VisItViewer.C

```diff
@@ -84,12 +87,18 @@ VisItViewer::Initialize(int *argc, char ***argv)
         {
             nowin = true;
         }
-#ifdef VISIT_OSPRAY
+#ifdef VISIT_OSPRAY // ospray 1.6.1, vtk 8
         else if (strcmp((*argv)[i], "-ospray") == 0)
         {
             debug5 << "Viewer launching with OSPRay" << endl;
             avtCallback::SetOSPRayMode(true);
         }
+#elif defined(HAVE_OSPRAY) // ospray 2.8, vtk 9
```

Comment:
Why not just...


```
         else if (strcmp((*argv)[i], "-ospray") == 0)
         {
             debug5 << "Viewer launching with OSPRay" << endl;
#ifdef VISIT_OSPRAY // ospray 1.6.1, vtk 8
             avtCallback::SetOSPRayMode(true);
 #elif defined(HAVE_OSPRAY) // ospray 2.8, vtk 9
             avtCallback::SetUseOSPRay(true);
#endif
         }
```

Reply:
Didn't think of it, should I make the change?

Reply:
I think this latter approach is a bit simpler to read, so yes.

Reply:
Actually, your suggested change has the if test for  '-ospray' option compiled even if VisIt wasn't built with ospray support.

Reply:
Doh :cry:!. But, we are just introspecting here, not printing anything in response. So, maybe...

         else if (strcmp((*argv)[i], "-ospray") == 0)
         {
             debug5 << "Viewer launching with OSPRay" << endl;
#ifdef VISIT_OSPRAY // ospray 1.6.1, vtk 8
             avtCallback::SetOSPRayMode(true);
 #elif defined(HAVE_OSPRAY) // ospray 2.8, vtk 9
             avtCallback::SetUseOSPRay(true);
#else
            std::cerr << "ospray not enabled in this build" << std::endl;
#endif
         }

I note that above, `-nowin` is always an option, but do all types of builds of VisIt support `-nowin`?